pkg/webui/lib/shared-messages.js

```diff
@@ -187,7 +187,7 @@ export default defineMessages({
   payloadFormattersUpdateFailure: 'There was an error updating the payload formatter',
   payloadFormattersUpdateSuccess: 'Payload formatter updated',
   payloadFormattersUplink: 'Uplink payload formatters',
-  phyVersion: 'PHY version',
+  phyVersion: 'Regional Parameters version',
```

Comment:
I'm a bit undecided whether we should change the message ID as well to something like `regionalParametersVersion`, but I think it's fine.

Reply:
```suggestion
  phyVersion: 'Regional parameters version',
```
Sentence case.

Reply:
yeah I agree it is fine, as this is mostly for the user clarity

Reply:
"Regional Parameters" are a spec, so that should be title case